### ISSUE TITLE
The second fire button only works after opening the GUI when using default.uae

### ISSUE THREAD

**Issue by Retro1968** [BlitterStudio/amiberry#986]
Raspberry Pi 4 with RetroPie 4.8. Amiberry 5.2 with AmigaOS 3.2.1

The game Mega Typhoon supports two fire buttons.
Sometimes the second fire button works in game, sometimes it doesn't.

In order for the second fire button to work properly again, I have to go into the amiberry GUI with F12 and exit it again immediately.

It was tested with amiberry 5.2 (2022-06-11), which I compiled with this command: make -j4 PLATFORM=rpi4-sdl2,
and the precompiled binaries of the RetroPie setup.

[default.zip](https://github.com/midwan/amiberry/files/8883779/default.zip)

Cheers
Retro1968

P.S.: Since I can't think of any other game that supports two fire buttons, I'm asking for information so I can test it with another game.

**Comment by midwan** [comment]
Thanks for reporting this - I haven't tried to recreate it yet, but from the description it sounds like some controller option(s) are not applied if the GUI is not opened once.
I'll look into it.

**Comment by Retro1968** [comment]
> @Retro1968 What options have you set in the Input panel, and specifically in Port 1 where your controller is? Have you changed it from the Default mode, for example? I'm trying to narrow down which option(s) are those that are not applied until you open the GUI, as you describe.

I sent you my default.uae and my amiberry.conf

[configs.zip](https://github.com/midwan/amiberry/files/8995607/configs.zip)

**Comment by Retro1968** [comment]
> @Retro1968 Thanks, that helps. Could you do one test please? What if you rename your `default.uae` to something else, and load it when the emulator starts up instead? Perhaps this only happens when using a `default.uae` file, and not with normal configs otherwise. This one is a little special, since it's auto-loaded on startup.

No problem, I'm happy to help. :-)

If I rename the default.uae, I get to the GUI first when I start amiberry. After I loaded the renamed default.uae from there, the second fire button works. I tested this with Mega Typhoon.

Cheers
Retro1968



PS.: I would like to draw your attention to this issue again, since you have not written anything about it yet.
https://github.com/BlitterStudio/amiberry/issues/985

The guru messages have been appearing there since version amiberry 5.2 (2022-06-11). A version before everything ran error-free. So the error would probably be easy to fix since you would only have to look at what changes you made to amiberry.
